src/test/ui/associated-const/defaults-cyclic-fail.stderr

```diff
@@ -1,30 +1,42 @@
-error[E0391]: cycle detected when const-evaluating + checking `Tr::A`
-  --> $DIR/defaults-cyclic-fail.rs:5:5
+error[E0391]: cycle detected when normalizing `<() as Tr>::A`
```

Comment:
I don't think there's a problem with this, btw, not everything has to have a `Span`.
The important thing is that the const-eval queries listed below are shown.